### PR TITLE
New version: Microsoft.PowerShell version 7.3.6.0

### PR DESCRIPTION
**Pull request by CoolPlayLin**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.9.0 :rocket:

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
# Please block this PR
The publisher appears they haven't added arm & arm64 version now, This PR shouldn't be merged before the architecture has been published

- Release URL: https://github.com/PowerShell/PowerShell/releases/tag/v7.3.6

**Comment by stephengillie** [comment]
What if: This PR has passed manual validation, and could be merged. This specific package would need approval too.

**Comment by mdanish-kh** [comment]
@stephengillie We can take this PR, it doesn't seem to be a blocking issue since the PowerShell team releases the MSIX after a few days of the initial publish of the release on GitHub. The msix are later added in a separate PR. cc @anamnavi to confirm this as well please

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml:16]:
```suggestion
```
Not needed now to avoid messing up package correlation, can be added later when msix are released

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.locale.en-US.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.locale.en-US.yaml:75]:
```suggestion
ReleaseNotes: |-

  Build and Packaging Improvements

  Bump .NET to 7.0.306
    - Update Notices file
    - Don't publish notice on failure because it prevents retry
    - Bump .NET to 7.0.306 (#19945)
    - Remove the property disabling optimization (#19952)
    - Add ProductCode in registry for MSI install (#19951)
    - Update variable used to bypass the blocking check for multiple NuGet feeds (#19953)
    - Change System.Security.AccessControl preview version to stable version (#19931)

  Documentation and Help Content
    - Update the link for getting started in README.md (#19947)
```

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml:28]:
```suggestion
  ProductCode: '{3E1D544C-E761-430F-B278-43E9FBFF0E4F}'
```

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml:23]:
```suggestion
  ProductCode: '{02A236F6-A7AA-4B3C-81F7-1A1B781616E3}'
```

**Comment by anamnavi** [comment]
you can approve this. I'll create a separate PR within the hour for the msix installers for this release.

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml:20]:
```suggestion
```

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml:25]:
```suggestion
```

File: manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml:8]:
```suggestion
Scope: machine
InstallerType: wix
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by anamnavi** [comment]
@mdanish-kh if it's easier I can create the PR for this whole version (msi and msix). I just need 30 mins or so, just let me know if it would be better

**Comment by mdanish-kh** [comment]
> @mdanish-kh if it's easier I can create the PR for this whole version (msi and msix). I just need 30 mins or so, just let me know if it would be better

That sounds better, I'll close this in favor of the new one when you create it

**Comment by mdanish-kh** [comment]
Close with reason: Replaced by #112154;

**Pull request closed without merging**
